src/handlers/mod.rs

```diff
@@ -650,17 +628,6 @@ pub async fn mirror_blob_handler(
         }
     }
 
-    match extract_file_hash_from_auth_event(&auth_event) {
```

Comment:
As the result is unused i don't fully understand what this was supposed to do, so i removed it, let me know if it does serve some function i'm missing.